### PR TITLE
Service Bus ARM template for live tests

### LINKED ISSUE #9882: Create ARM template for SB live tests
* ARM template should be created to deploy and clean up Service Bus resources after our nightly tests.
* Delete persistent Service Bus resource keys in KeyVault.

### PR DESCRIPTION
**Pull request by conniey**
* Adds Service Bus ARM template for live tests
* Adds indentation of JSON to two spaces rather than default four spaces. (ARM templates have many nested objects, it becomes unreadable.

Fixes #9882

### CODE REVIEW

File: sdk/servicebus/test-resources.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    }
+  },
+  "variables": {
+    "authRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', variables('namespaceName'), variables('sharedAccessKeyName'))]",
+    "contributorRoleId": "b24988ac-6180-42a0-ab88-20f7382dd24c",
+    "namespaceName": "[concat('sb-java-', parameters('baseName'))]",
+    "namespaceDomainNameSuffix": "servicebus.windows.net",
+    "queueName": "[concat('queue-', parameters('baseName'))]",
+    "subscriptionName": "[concat('subscription-', parameters('baseName'))]",
+    "serviceBusDataOwnerRoleId": "090c5cfd-751d-490a-894a-3ce6f1109419",
```
Comment (hemanttanwar) [inline sdk/servicebus/test-resources.json:59]: Do we also need to give Sender and Receiver role ? https://docs.microsoft.com/en-us/azure/service-bus-messaging/authenticate-application#built-in-rbac-roles-for-azure-service-bus

File: .editorconfig [hunk 2]
```diff
@@ -10,6 +10,9 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 max_line_length = 120
 
+[*.json]
```
Comment (hemanttanwar) [inline .editorconfig:13]: is this file intentional ?

Comment (conniey) [reply .editorconfig:13]: Yes. 

> Adds indentation of JSON to two spaces rather than default four spaces. (ARM templates have many nested objects, it becomes unreadable.

File: sdk/servicebus/test-resources.json (hunk 1 above)
Comment (conniey) [reply sdk/servicebus/test-resources.json:59]: I have it in Line 120
